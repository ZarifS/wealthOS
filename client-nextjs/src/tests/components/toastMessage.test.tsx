import { render } from '@testing-library/react';
import ToastMessage from '../../components/toastMessage';

describe('ToastMessage component', () => {
  it('renders correctly', () => {
    const { asFragment } = render(
      <ToastMessage message="This is a success message" state="success" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
