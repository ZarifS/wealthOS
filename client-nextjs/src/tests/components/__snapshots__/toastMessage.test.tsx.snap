// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastMessage component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="toast success"
  >
    This is a success message
    <button
      class="closeButton"
    >
      Ã—
    </button>
  </div>
</DocumentFragment>
`;
